lib/snapshot/index.js

```diff
@@ -36,15 +37,35 @@ function getVulnerableFunctionsLocations(moduleInfo) {
   });
 
   try {
-    return ast.findAllVulnerableFunctionsInScript(
+    const vulnerableFunctionsFound = ast.findAllVulnerableFunctionsInScript(
       fs.readFileSync(scriptPath), vulnerableFunctionNames);
+    const foundFunctionNames = Object.keys(vulnerableFunctionsFound);
+    validateFoundFunctions(scriptPathFunctions, moduleInfo, vulnerableFunctionNames, foundFunctionNames);
+    return vulnerableFunctionsFound;
   } catch (error) {
     debug(`Error finding vulnerable methods ${vulnerableFunctionNames}` +
       ` in script path ${scriptPath}: ${error}`);
     return {};
   }
 }
 
+function validateFoundFunctions(scriptPathFunctions, moduleInfo, functionsExpected, functionsActual) {
+  const functionsNotFound = functionsExpected.filter(f => !functionsActual.includes(f));
+  if (functionsNotFound && functionsNotFound.length > 0) {
+    const warningEvent = {
```

Comment:
Can you put an example event here? Just to see what information is sent

Reply:
```
snyk:nodejs:transmitter Event added to transmission queue: {"warning":{"message":"instrumentation discrepacy: missing functions from source code","moduleInfo":{"version":"1.2.11","name":"mime","baseDir":"/Users/amir/projects/nodejs-runtime-agent/node_modules/st/node_modules/mime","scriptRelativePath":"mime.js","scriptPath":"/Users/amir/projects/nodejs-runtime-agent/node_modules/st/node_modules/mime/mime.js"},"snapshotInfo":[{"name":"Mime.prototype.shlookup","semver":["<1.4.1"]}],"functionsInfo":{"functionsExpected":["Mime.prototype.shlookup"],"functionsActual":[],"functionsNotFound":["Mime.prototype.shlookup"]}},"timestamp":"2019-01-21T14:27:47.978Z"}
```